core/assets/bundles/bundle_vi.properties

```diff
@@ -1074,38 +1074,38 @@ block.shale-wall.name = Tường đá phiến sét
 block.scrap-wall.name = Tường phế liệu
 block.scrap-wall-large.name = Tường phế liệu lớn
 block.scrap-wall-huge.name = Tường phế liệu khổng lồ
-block.scrap-wall-gigantic.name = Tường phế liệu siêu khổng lồ
-block.thruster.name = Thruster
-block.kiln.name = Lò nung
+block.scrap-wall-gigantic.name = Tường phế liệu siêu to khổng lồ
+block.thruster.name = Động cơ phản lực
+block.kiln.name = Lò nung kính
 block.graphite-press.name = Máy nén than chì
 block.multi-press.name = Máy nén than chì lớn
 block.constructing = {0} [lightgray](Đang xây dựng)
-block.spawn.name = Khu vực tạo ra kẻ địch
-block.core-shard.name = Căn cứ: Cơ sở
-block.core-foundation.name = Căn cứ: Trụ sở
-block.core-nucleus.name = Căn cứ: Trung tâm
+block.spawn.name = Khu vực triêu hồi kẻ địch
+block.core-shard.name = Căn cứ: Mảnh
+block.core-foundation.name = Căn cứ: Nền Móng
+block.core-nucleus.name = Căn cứ: Hạch Tử
 block.deepwater.name = Nước sâu
 block.water.name = Nước
-block.tainted-water.name = Nước nhiểm bẩn
-block.darksand-tainted-water.name = Nước nhiễm bẩn cát đen
+block.tainted-water.name = Nước ô uế
```

Comment:
Orginal is enough to understand.
you make it harder to understand what it mean.